src/assets/i18n/en.json

```diff
@@ -83,12 +83,40 @@
       }
     },
     "DROPBOX": {
+      "C": {
```

Comment:
I wonder where these are all coming from. Could you please remove all those empty translations?

Reply:
BabelEdit was adding them, usually I trust it to do the right thing. I remove it it though and just add the new key.